src/main/java/uk/gov/companieshouse/api/accounts/model/rest/Costs.java

```diff
@@ -6,14 +6,14 @@
 public class Costs {
 
     @JsonProperty("costs")
-    private Map<String, Cost> costs;
+    private Map<String, Cost> costsList;
```

Comment:
Would costsMap be better here? It is not a list.

Reply:
That's a fair point. I'll update